uiowa/uiowa
# Hotfix: Update Drush local aliases root path and fix inconsistent app aliases.

Resolves #4573 

# How to test

- On main, try rsyncing files like ` ddev drush rsync @site.prod:%files @site.local:%files`.
- See that it fails.
- Check out this branch. Do it again.
- Success.
